src/sentry/integrations/utils/code_mapping.py

```diff
@@ -140,14 +140,36 @@ def list_file_matches(self, frame_filename: FrameFilename) -> list[dict[str, str
             ]
 
             for file in matches:
-                stacktrace_root, source_path = find_roots(frame_filename.raw_path, file)
+                stack_path = frame_filename.raw_path
+                source_path = file
+
+                try:
+                    stack_root, source_root = find_roots(stack_path, source_path)
+                except Exception:
+                    logger.info(
+                        "Unexpected format for stack_path or source_path",
+                        extra={"stack_path": stack_path, "source_path": source_path},
+                    )
```

Comment:
do we really want to try and continue this code if there is an error? should we be returning / continuing?

Reply:
Good catch - fixing this now